js/binance.js

```diff
@@ -1903,6 +1984,29 @@ module.exports = class binance extends Exchange {
         return result;
     }
 
+    setMarkets (markets, currencies = undefined) {
+        super.setMarkets (markets, currencies);
+        const values = Object.values (this.markets);
+        let hasSpot = false;
+        for (let i = 0; i < values.length; i++) {
+            const entry = values[i];
+            hasSpot = hasSpot || entry['spot'];
+            if (hasSpot) {
+                break;
+            }
+        }
+        if (!hasSpot) {
```

Comment:
@frosty00  we have to disable this check for binanceusdm and binancecoinm
![image](https://user-images.githubusercontent.com/43336371/210061937-f1f67c50-7448-4c80-bbb3-88a3fd7cee3b.png)